### PR TITLE
Add a functional comment feature

### PR DESCRIPTION
**Pull request by niuamy**
Added a post request and created a comment class. The idea is for the user to be able to input a message into a text box, and on submit, the message will be displayed underneath the textbox.
Screenshot 2020-06-02 at 10 22 07 AM - Display 1

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Comments.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Google LLC
```
Comment (ashley3) [inline portfolio/src/main/java/com/google/sps/data/Comments.java:1]: nit: 2020

Comment (niuamy) [reply portfolio/src/main/java/com/google/sps/data/Comments.java:1]: I'm not sure if this was a mistake, but in the example files, it has the copyright being 2020?

Comment (niuamy) [reply portfolio/src/main/java/com/google/sps/data/Comments.java:1]: 2019*

Comment (ashley3) [reply portfolio/src/main/java/com/google/sps/data/Comments.java:1]: Ah in that case it's probably fine to leave as is

**Pull request merged**